composer.json

```diff
@@ -1,14 +1,13 @@
 {
-    "name": "sy/contact",
+    "name": "quazz/contact",
     "description": "N/A",
     "license": [
         "OSL-3.0"
     ],
     "require": {
-        "php": "~5.6.0|7.0.2|7.0.4|~7.0.6|~7.1.0"
+        "php": "~5.6.0|7.0.2|7.0.4|~7.0.6|~7.1.0|~7.2.0"
```

Comment:
we might remove PHP requirements altogether and depend on the specific magento core modules